test/unit_test/rag/test_naive_merge.py

```
@@ -215,3 +215,82 @@ def test_images_distinct_lazyimages_are_concatenated():
+    assert any("chunk_token_num" in r.message and "10" in r.message for r in caplog.records), [r.message for r in caplog.records]
+
+
+@pytest.mark.p0
+def test_hard_cap_with_images_single_oversized_segment_emitted_as_one_chunk(caplog):
+    """Mirror of the previous test for naive_merge_with_images: a single
+    oversized segment is emitted as one chunk and a warning is logged."""
+    import logging
+
+    huge = " ".join(["w" + str(i) for i in range(50)])  # 50 tokens
+    with caplog.at_level(logging.WARNING, logger="rag.nlp"):
+        chunks, imgs = naive_merge_with_images([huge], [None], chunk_token_num=10, delimiter=DEFAULT_DELIMITER)
```

Ruff (0.15.21)

[warning] Unpacked variable `imgs` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)